ui/app/css/itcss/components/tooltip.scss

```
@@ -1,11 +1,9 @@
+@import '../../../../../node_modules/react-tippy/dist/tippy.css';
```

This isn't supposed to have a file extension:
```suggestion
@import '../../../../../node_modules/react-tippy/dist/tippy';
```

Omitting the file extension instructs Sass to treat this as a Sass import, rather than a CSS import.